### DISCUSSION TITLE
Split view browsing

### DISCUSSION THREAD

**Discussion by add-n2x** [spacedriveapp/spacedrive#1885] [Feature Request]
I'm missing the old, shiny days of Norton Commander. I am wondering why currently no modern OS provides a traditional split view natively.

Please add a split view feature to Spacedrive.

This enables users to browse two locations at the same time, side by side. IMO it radically simplifies any file organization workflow a lot.

**Comment by utkubakir** [comment]
We have tabs (albeit very hidden and you can only open it with the shortcut) so split view sounds redundant. Would there be any use cases where you can't do the same with tabs?

**Reply by Airradda** [reply]
A common workflow for me using split view is having my downloads folder and the top level my document folder, which has folders for categories, open. This allows me to quickly drag and drop the files from the downloads folder as opposed to selecting the file, cutting, switching tab, selecting the category folder, and pasting. Not all file mangers let me paste into folder by simply selecting it and hitting paste, requiring me to have to open the category and then having to return the top level documents folder. 

TLDR:
Drag and Drop vs Select File->Cut File->Switch Tab->Select/Open Folder->Paste File

**Reply by utkubakir** [reply]
Woops forgot to reply, we talked this internally with the team and we will be adding split view in the future as well.

P.S. You can drag files (to the tab you want) to switch tabs as of now :)

**Reply by ketonik** [reply]
Cocoatechs' Path Finder has been doing this for over 20 years, Forklift has been doing this for over a decade, it's called Dual Pane Mode, sure not 'Native OS.

However, MacOS Finder's 'List View' since the 1980's can display all files in all sub folders in the same window so while it can't display folders and files on two different storage devices, it still goes a long way in aiding the user working inside sub folders of the same storage device, make's Microsoft's 1990's Tree View look stupid and utterly useless.
